lib/engaging_networks_rest/client/pages.rb

```diff
@@ -15,7 +15,12 @@ def page(page_id:)
       end
 
       def process_page_request(page_id:, supporter_data:)
```

Comment:
I think this is the wrong approach. I think we need to return the full response object instead of just and ID. 

That way, the consumer can decide what to do. We're also throwing away the supporterId and just returning what looks like a page action processing id....